modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -128,3 +128,76 @@ test('shows outside pages in path analysis', async ({apiHelpers, page}) => {
 
 	await apiHelpers.jsonWebServicesLayout.deleteLayout(String(sitePage.id));
 });
+
+test('shows tracked pages in path analysis', async ({apiHelpers, page}) => {
+	const pageTitle1 = 'My Page 1';
+
+	const sitePage1 = await createSitePage(apiHelpers, pageTitle1);
+
+	const pageTitle2 = 'My Page 2';
+
+	const sitePage2 = await createSitePage(apiHelpers, pageTitle2);
+
+	const channelName = 'My Property - ' + getRandomString();
+
+	await syncAnalyticsCloud(page, channelName);
+
+	await page.waitForTimeout(10000);
+
+	await page.getByText(pageTitle1).first().click();
```

Comment:
Hey @rachaelkoestartyo, This navigation doesn't appear correct to me. Here you are trying to find the element with the name of your created page and trying to click to access the page, but you are currently on the AC connection page with DXP and this element with your page does not appear in the UI.

My suggestion is that you create this function below. On that you can pass to the function the name of the page that you want to access (site name also if you had another site created) and your page will be loaded directly in the URL as a direct access to the page that was passed.

After that, you can use this `page.getByText` function to access page 2 as a reference page. What do you think?


```
export async function navigateToSitePage(page, siteName, pageName) {
    // Replace spaces with hyphens and convert to lowercase
    let pageNameURL = pageName.replace(/ /g, "-").toLowerCase();

    if (siteName) {
        // Replace spaces with hyphens and convert to lowercase
        let siteNameURL = siteName.replace(/ /g, "-").toLowerCase();

        // Construct URL and navigate, considering that a new site is already created in the instance
        await page.goto(`${liferayConfig.environment.baseUrl}/web/${siteNameURL}/${pageNameURL}`);
    } else {
        // Construct URL and navigate, considering the site is the default for Liferay DXP
        await page.goto(`${liferayConfig.environment.baseUrl}/${pageNameURL}`);
    }
}
```